code/game/objects/items/devices/PDA/cart.dm

```diff
@@ -373,6 +378,60 @@ Code:
 						menu += " [S[A.equipment+1]] [S[A.lighting+1]] [S[A.environ+1]] [add_lspace(A.lastused_total, 6)]  [A.cell ? "[add_lspace(round(A.cell.percent()), 3)]% [chg[A.charging+1]]" : "  N/C"]<BR>"
 
 				menu += "</FONT></PRE>"
+		if (53)
+			menu = "<h4><img src=pda_alert.png> Please select an Alert Computer</h4><BR>"
+			alertmonitor = null
+			alertmonitors = list()
+			var/alertcount = 0
+
+
+			for(var/obj/machinery/computer/station_alert/aMon in world)
+				if(!(aMon.stat & (NOPOWER|BROKEN)))
+					var/turf/T = get_turf(src)
+					if(T.z == aMon.z)//the application may only detect power monitoring computers on its Z-level.
+						alertcount++
+						alertmonitors += aMon
+
+
+			if(!alertcount)
+				menu += "No Alert Computer detected in the vicinity.<BR>"
+			else
+
+				menu += "<FONT SIZE=-1>"
+				var/count = 0
+				for(var/obj/machinery/computer/station_alert/aMon in alertmonitors)

```

Comment:
~~~
menu = "No Alert Computer detected in the vicinity"
var/found = 0
-loop-
    if(zlevel)
        if(!found)
              menu = "hreflink"
        else
              menu += "hreflink"
~~~